lib/CustomPropertyFilters/CustomPropertyFiltersForm.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+import { FieldArray } from 'react-final-form-arrays';
+
+import {
+  Button,
+  Col,
+  Label,
+  Layout,
+  Modal,
+  ModalFooter,
+  Row,
+  Select,
+} from '@folio/stripes/components';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+
+import EditCard from '../EditCard';
+import { required as requiredValidator } from '../validators';
+
+import CustomPropertyRule from './CustomPropertyRule';
+
+class CustomPropertyFiltersForm extends React.Component {
+  static propTypes = {
+    custPropName: PropTypes.string,
+    customProperties: PropTypes.arrayOf(
+      PropTypes.shape({
+        label: PropTypes.string.isRequired,
+        name: PropTypes.string.isRequired,
+        type: PropTypes.string.isRequired,
+      })
+    ),
+    filterHandlers: PropTypes.shape({
+      state: PropTypes.func.isRequired,
+    }),
+    form: PropTypes.shape({
+      change: PropTypes.func.isRequired,
+      mutators: PropTypes.shape({
+        push: PropTypes.func.isRequired,
+      }),
+    }),
+    handleSubmit: PropTypes.func.isRequired,
+    values: PropTypes.object,
+  };
+
+  state = {
+    editingFilters: false,
+  }
+
+  renderFooter = () => (
+    <ModalFooter>
+      <Button
+        buttonStyle="primary"
+        data-test-apply-filters
+        onClick={values => {
+          const promise = this.props.handleSubmit(values);
+          promise?.then(() => this.setState({ editingFilters: false })); // eslint-disable-line no-unused-expressions
+        }}
+      >
+        <FormattedMessage id="stripes-erm-components.apply" />
+      </Button>
+      <Button
+        data-test-cancel-filters
+        onClick={() => this.setState({ editingFilters: false })}
+      >
+        <FormattedMessage id="stripes-erm-components.cancel" />
+      </Button>
+    </ModalFooter>
+  );
+
+  render() {
+    const {
+      custPropName,
+      customProperties,
+      form: {
+        mutators: { push },
+        change,
+      },
+      values,
+    } = this.props;
+    const { editingFilters } = this.state;
+
+    return (
+      <>
+        <Button
+          data-test-open-customProperties-filters
```

Comment:
Looks like the tests are breaking because this data-test attribute is different from what's expected. Can you do a pass through this PR and make sure we're using the same format across all these data-test attrs and ids? Ie, `custom-properties` vs `custprops` vs `customProperties`?

Reply:
Yes. But I decided on `custprop` instead of `custom-properties` because almost all of the ids and attributes used this already